nemo_curator/modules/fuzzy_dedup.py

```diff
@@ -499,15 +506,26 @@ def __call__(self, dataset: DocumentDataset):
         if self.config.false_positive_check:
             # Map buckets to lower cardinality distribution
             print(f"Stage{stage_num} (False Positive Check): Starting Map_Buckets")
-            ddf_mapped_buckets_w_anchors = self.map_buckets.map_buckets_with_anchors(
-                documents_df=dataset.df, buckets_df=buckets_df.df
-            )
+            t0 = time.time()
             mapped_buckets_w_anchors_path = os.path.join(
                 self.config.cache_dir, "anchor_docs_with_bk.parquet"
             )
-            ddf_mapped_buckets_w_anchors.to_parquet(
-                mapped_buckets_w_anchors_path, write_index=False
+            with performance_report_if(
+                self.config.profile_dir,
+                f"map_buckets-{datetime.now().strftime('%Y%m%d_%H%M%S')}.html",
```

Comment:
Minor nit `%Y%m%d_%H%M%S` , maybe we change it to `PROFILE_TIMESTAMP_FORMAT` and change it to a constant.

Reply:
Made the change!